shared/utils.ts

```diff
@@ -194,3 +193,8 @@ export const debounceWithArgs = <T extends (...args: any[]) => any>(
     return t[k](...args);
   };
 };
+
+export const convertBytesToGB = (maxFileSize: number) =>
+  maxFileSize / 1024 / 1024 / 1024;
+export const covertGBToBytes = (maxFileSize: number) =>
```

Comment:
```suggestion
export const convertGBToBytes = (maxFileSize: number) =>
```
Do not forget to update it in other places.